docs/misc/openindiana.md

```diff
@@ -98,10 +98,10 @@ OpenIndiana contains the following enterprise class features and more:
 | COMSTAR |[Common Multiprotocol SCSI TARget - (ISCSI Target Framework)](http://illumos.org/man/1m/itadm)
 | KVM | [Kernel Virtual Machine - (Operating System Virtualization)](https://en.wikipedia.org/wiki/Kernel-based_Virtual_Machine)
 | Zones | [OS Level Virtualized Application Containers](https://en.wikipedia.org/wiki/Solaris_Containers)
-| Time-Slider | [Automated ZFS Snapshots and Rollbacks](http://www.serverwatch.com/tutorials/article.php/3831881/Say-Cheese-OpenSolaris-Time-Slider.htm)
-| RBAC | [Role-Based Access Control](http://www.c0t0d0s0.org/archives/4073-Less-known-Solaris-features-RBAC-and-Privileges-Part-1-Introduction.html)
-| IPMP | [IP Network Multipathing](http://www.c0t0d0s0.org/archives/6292-Less-known-Solaris-features-IP-Multipathing-Part-1-Introduction.html)
-| DLMP | [Data Link Multipathing](http://www.c0t0d0s0.org/archives/7553-Less-known-Solaris-Features-Data-Link-Multipathing.html)
+| Time-Slider | [Automated ZFS Snapshots and Rollbacks](https://web.archive.org/web/20160420090032/http://www.serverwatch.com/tutorials/article.php/3831881/Say-Cheese-OpenSolaris-Time-Slider.htm)
+| RBAC | [Role-Based Access Control](https://web.archive.org/web/20170910053725/http://www.c0t0d0s0.org/archives/4073-Less-known-Solaris-features-RBAC-and-Privileges-Part-1-Introduction.html)
```

Comment:
At least RBAC description is present in handbook (and in https://wiki.openindiana.org/oi/pfexec+and+profiles+shortly+explained ). Refering to wayback machine in 'about' page feels bad, at least when it's not unnecessary. 
It's a shame that wiki has 0 pages describing time slider...

Reply:
Indeed, the same is true for some of the other links which are to very old websites which have themselves been offline for 5+ years.
The next step is certainly to pull in content from the wiki and these links, check it's all up to date and still relevant to current oi and then remove the old links.

Reply:
As for links to http://www.c0t0d0s0.org/, content still exists, but links are different, for example, http://www.c0t0d0s0.org/2010-01-21/less-known-solaris-features-ip-multipathing-part-1-introduction.c0t0d0s0 .
Be careful to avoid copy-pasting data from external sources due to copyright restrictions. But you can always contact authors if they agree for you to publish some extracts on our site.